Sources/TensorFlow/Operators/NN.swift

```diff
@@ -820,3 +820,51 @@ func _vjpAvgPool3D<Scalar: TensorFlowFloatingPoint>(
     }
   )
 }
+
+//===------------------------------------------------------------------------------------------===//
+// Rearrange depth/space
+//===------------------------------------------------------------------------------------------===//
+
+/// Returns `input` rearranged from depth into blocks of spatial data.
+///
+/// - Parameters:
+///   - input: The tensor to be rearranged.
+///   - blockSize: The size of the spatial block.
+/// - Precondition: The input must have rank 4.
+/// - Precondition: The block size must be greater than 1.
+@differentiable(wrt: input where Scalar: TensorFlowFloatingPoint)
+public func depthToSpace<Scalar>(_ input: Tensor<Scalar>, blockSize: Int) -> Tensor<Scalar> {
```

Comment:
This is an example where almost everything you need to know can be captured in the summary and a `- Parameters:` block just makes it harder to assimilate the meaning of the function

```suggestion
/// Returns `input` rearranged from depth into spatial data blocks of size `b`.
///
/// - Precondition: `input.rank == 4 && b > 1`.
@differentiable(wrt: input where Scalar: TensorFlowFloatingPoint)
public func depthToSpace<Scalar>(_ input: Tensor<Scalar>, blockSize b: Int) -> Tensor<Scalar> {
```

Reply:
I updated the comment:
> Returns `input` rearranged from depth into blocks of spatial data of specified block size.

https://github.com/tensorflow/swift-apis/pull/710/commits/ecc981e235577162e538ad04e2f88a174497f315#diff-3aee58937ce4e8fc2ea9fcdbd5266e93R828
How about it?

Reply:
Apple's Developer Publications imposed the “of the specified …” style  on all the doc comments we wrote in the Swift Standard Library to say, removing our direct references to parameter names  (actually the DevPubs guidelines also forbid `input` to be written in “code voice”), but it's actually suboptimal both for the person writing the comment (sometimes there's more than one of a thing in the parameter list and you have to come up with creative ways to refer to each one) and the person reading it (who has to mentally work around those creative and longwinded phrasings and associate the phrases with actual parameters).  Sometimes, the lack of clarity must be compensated for by adding a `- Parameters:` clause to the documentation, resulting in long and tedious descriptions of simple APIs whose meaning could be taken in at a glance. 

The parameters have names; let's use them, please.

Reply:
Sorry, I forgot the <rant on> <rant off> delimiters ;-)

Reply:
> The parameters have names; let's use them, please.

OK.
Please send same suggestion again. I don't want to steal your credit.

Reply:
Please don't worry about the credit, and if somebody else bothers you about it feel free to send them after me with blame ;-)

Reply:
OK. Updated with small change.
https://github.com/tensorflow/swift-apis/pull/710/commits/47a31206b07cf2f23e472e96db4d98e63694cb0f
Thank you for your help.

Reply:
LGTM, and thank you for your patience with all my very particular requests.